honeybee_radiance/cli/translate.py

```diff
@@ -159,6 +185,9 @@ def model_to_rad_folder(
             model, folder, config_file, minimal, views=view, grids=grid,
             full_match=full_match
         )
+
+        model.to_hbjson('output_model', '.')
```

Comment:
It's probably only a couple of seconds to re-write this model, but I suggest doing this only when there has been a change to the model. Otherwise, we can simply rename/copy the input file.

Reply:
I changed this so it will simply copy the file.